### PR TITLE
drivers: video: stm32_dcmi: add set and get ctrl

### PR DESCRIPTION
**Pull request by CharlesDias**
Add set_ctrl and get_ctrl functions. These were tested with https://github.com/zephyrproject-rtos/zephyr/pull/71463

### CODE REVIEW

File: drivers/video/video_stm32_dcmi.c [hunk 1]
```diff
@@ -356,6 +356,28 @@ static int video_stm32_dcmi_get_caps(const struct device *dev,
 	return ret;
 }
 
+static inline int video_stm32_dcmi_set_ctrl(const struct device *dev, unsigned int cid, void *value)
+{
+	const struct video_stm32_dcmi_config *config = dev->config;
+	int ret = -ENOTSUP;
```
Comment (erwango) [inline drivers/video/video_stm32_dcmi.c:362]: ret initialization not required

File: drivers/video/video_stm32_dcmi.c [hunk 2]
```diff
@@ -356,6 +356,28 @@ static int video_stm32_dcmi_get_caps(const struct device *dev,
...
+	/* Forward to source dev if any */
+	ret = video_set_ctrl(config->sensor_dev, cid, value);
+
+	return ret;
+}
+
+static inline int video_stm32_dcmi_get_ctrl(const struct device *dev, unsigned int cid, void *value)
+{
+	const struct video_stm32_dcmi_config *config = dev->config;
+	int ret = -ENOTSUP;
```
Comment (erwango) [inline drivers/video/video_stm32_dcmi.c:373]: Ditto

**Review by erwango — COMMENTED** [review]
Thanks @CharlesDias !
One minor comment, otherwise LGTM

File: drivers/video/video_stm32_dcmi.c (hunk 2 above)
Comment (CharlesDias) [reply drivers/video/video_stm32_dcmi.c]: Done.

**Review by erwango — APPROVED** [review]

**Review by marwaiehm-st — APPROVED** [review]
LGTM

**Pull request merged**